src/Libraries/PythonNodeModels/Properties/Resources.en-US.resx

```diff
@@ -182,13 +182,13 @@
     <value>Learn about Python3 updates.</value>
   </data>
   <data name="PythonScriptEditorRevertButtonTooltip" xml:space="preserve">
-    <value>Revert the changes made to current Python node editor.</value>
+    <value>Revert the changes made to current Python node editor and close it.</value>
   </data>
   <data name="PythonScriptEditorRunButtonTooltip" xml:space="preserve">
-    <value>Run Python node with current code in editor.</value>
+    <value>Run Python node with current code in editor and save it.</value>
```

Comment:
For some reason, this reads backwards to me, - what do you think about - `Save current code and run it`?

Reply:
Will do